src/utils/internal.ts

```diff
@@ -1412,7 +1412,7 @@ function assertExpectedSpecVersion(
  *
  * Checks SQ version compatibility with the SDK
  */
-export async function assertExpectedSqVersion(context: Context): Promise<void> {
```

Comment:
This is only warning now, so I changed the name. `assert` makes it feel like it should throw to me